portfolio/src/main/webapp/index.html

```diff
@@ -1,27 +1,183 @@
+                  <div class="subheading mb-3">Intern</div>
+                  <p>Developed scripts using Python and Bash for processing large amounts of MRI pipeline data. Communicated with end-users to improve and troubleshoot code while resolving technical problems in a timely manner. Built a database for the CU's Got Brains study using REDCap.</p>
+               </div>
+               <div class="flex-shrink-0"><span class="text-primary">May 2019 - August 2019</span></div>
+            </div>
+         </div>
+      </section>
+      <hr class="m-0" />
+      <!-- Education-->
+      <section class="resume-section" id="education">
+         <div class="resume-section-content">
+            <h2 class="mb-5">Education</h2>
+            <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
+               <div class="flex-grow-1">
+                  <h3 class="mb-0">Vanderbilt University</h3>
+                  <div class="subheading mb-3">Bachelor of Science</div>
+                  <div>Computer Science and Applied Mathematics</div>
+                  <div>Minor in Engineering Management</div>
+               </div>
+               <div class="flex-shrink-0"><span class="text-primary">August 2018 - May 2022</span></div>
+            </div>
+            <div class="d-flex flex-column flex-md-row justify-content-between">
+               <div class="flex-grow-1">
+                  <h3 class="mb-0">Relevant Coursework</h3>
+                  <ul class="fa-ul mb-0">
+                     <li>
+                        <span class="fa-li"><i class="fas fa-check"></i></span>Intermediate Software Design in C++ (CS 3251)
+                     </li>
+                     <li>
+                        <span class="fa-li"><i class="fas fa-check"></i></span>Program Design and Data Structures in C++ (CS 2201)
+                     </li>
+                     <li>
+                        <span class="fa-li"><i class="fas fa-check"></i></span>Programming and Problem Solving in Java (CS 1101)
+                     </li>
+                     <li>
+                        <span class="fa-li"><i class="fas fa-check"></i></span>Digital Systems (EECE 2123)
+                     </li>
+                     <li>
+                        <span class="fa-li"><i class="fas fa-check"></i></span>Discrete Structures (CS 2212)
+                     </li>
+                     <li>
+                        <span class="fa-li"><i class="fas fa-check"></i></span>Methods of Linear Algebra (MATH 2410)
+                     </li>
+                  </ul>
+               </div>
+            </div>
+      </section>
+      <hr class="m-0" />
+      <!-- Skills-->
+      <section class="resume-section" id="skills">
+      <div class="resume-section-content">
+      <h2 class="mb-5">Skills</h2>
+      <div class="subheading mb-3">Proficient in:</div>
+      <ul>
+      <li>Java</li>
```

Comment:
could you indent the li elements?

https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

Reply:
yep, done!